src/StackedTableCell.js

```diff
@@ -0,0 +1,100 @@
+import React, { Fragment } from 'react'
+import css from 'styled-jsx/css'
+import propTypes from 'prop-types'
+
+import { Consumer } from './StackedTable/TableContext.js'
+
+const tableCellStyles = css`
+    td {
+        border-bottom: 1px solid #e8edf2;
+        padding: 0 12px;
+        font-size: 14px;
+        width: 100%;
+        display: block;
+    }
+`
+
+const ContentWithTitle = ({ title, children }) => (
+    <Fragment>
+        {title && <span className="title">{title}</span>}
+        <span className="content">{children}</span>
+
+        <style jsx>{`
+            .title {
+                display: block;
+                white-space: normal;
+                min-height: 24px;
+                line-height: 18px;
+                padding: 8px 0 0 0;
+                font-weight: bold;
+                white-space: nowrap;
```

Comment:
Right now the cell title appears more important than the content itself, I think this is the wrong way around. The content is important and the cell title is just a reference. I suggest adjusting the adding/adjusting the following styles for the title :
```
font-size: 13px;
line-height: 16px;
font-weight: 400;
padding: 8px 0 4px 8px;
color: ${colors.grey700};
```

Reply:
Sure can do, but the padding on the left looks very weird, breaks the vertical optical line.
Would `paddding: 8px 0 4px` be ok as well?

Reply:
Apologies, my mistake. `padding: 8px 0 4px` is correct, yes.